sampleproject/components/calendar/calendar.py

```diff
@@ -23,3 +23,27 @@ def get(self, request, *args, **kwargs):
     class Media:
         css = "calendar/calendar.css"
         js = "calendar/calendar.js"
+
+
+@component.register("calendar_relative")
+class CalendarRelative(component.Component):
```

Comment:
I've added a variant of the Calendar component to show how to use it with relative paths

Reply:
Excellent!